keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -2547,7 +2547,7 @@
           "type": "array",
           "items": {
             "type": "string",
-            "description": "JSON web key operations. For more information, see JsonWebKeyOperation.",
+            "description": "JSON web key operations. The default is all of the operation values; encrypt, decrypt, sign, verify, wrapKey and unwrapKey. For more information, see JsonWebKeyOperation.",
```

Comment:
If there is a default value, we should provide a value for the default property instead of just putting it in the description. And we shouldn't duplicate default values between the actual default value property and the description since that causes problems for code generators and applications that want to provide the *actual* generated name instead of the name in swagger.